index.html

```diff
@@ -6,8 +6,20 @@
     <title>Resque_a_pet</title>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/fonts.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/main.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/heading.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
 </head>
 <body>
-    
+ <div class="container">
+  <header>
+     <img class="logo-header" src="assets/images/logo.png"/>
+     <img class="photo-header" src="assets/images/the-dog-white-and-brown.jpeg"/>   
```

Comment:
Tego typu rzeczy robi się teraz inaczej. Na pewno tego używałeś wcześniej.
Zamiast img robimy to na background-image.

Dodatkowo temat jak wyżej: header-photo.

Reply:
Poprawione